_data/tools.json

```diff
@@ -45,6 +45,15 @@
       "type": "SAST"
        },
        {
+      "title": "Port Scanner Online ",
+      "url": "https://portscanner.online/",
+      "owner": "MUNSIRADO",
+      "license": "Commercial or Free",
+      "platforms": "SaaS / Plugins",
+      "note": "Online DAST tool for scanning open ports using NMAP, Unicornscan, and RustScan. It provides fast, accurate, and comprehensive port scanning directly from your browser",
+      "type": "DAST"
```

Comment:
Port scanners are not usually considered to be DAST.
I dont think this tool belongs on this list...

Reply:
Agreed